tutorials/SDF/car_demo.md

```diff
@@ -0,0 +1,520 @@
+
+We need to connect these links together; here comes the job of `<joint>` tag.
+The joint tag connects two links together and defines how they will move with respect to each other.
+Inside the `<joint>` tag we need to define the two links to connect and their relations(way of movement).
+
+#### Left wheel joint
+
+```xml
+<joint name='left_wheel_joint' type='revolute'> <!--continuos joint is not supported yet-->
+    <pose relative_to='left_wheel'/>
+```
+
+Our first joint is the `left_wheel_joint`.
+It takes two attributes: the name `name='left_wheel_joint'` and the type `type='revolute'`.
+`revolute` type gives 1 rotational degree of freedom with joint limits.
+The pose of the joint is the same as `left_wheel` frame.
+
+```xml
+    <parent>chassis</parent>
+    <child>left_wheel</child>
+```
+
+Every joint connects two links (bodies) together.
+Here we connect the `chassis` with the `left_wheel`.
+`chassis` is the parent link and `left_wheel` is the child link.
+
+```xml
+    <axis>
+        <xyz expressed_in='__model__'>0 1 0</xyz> <!--can be described to any frame or even arbitrary frames-->
+        <limit>
+            <lower>-1.79769e+308</lower>    <!--negative infinity-->
+            <upper>1.79769e+308</upper>     <!--positive infinity-->
+        </limit>
+    </axis>
+</joint>
+```
+
+Here we define the axis of rotation.
+The axis of rotation can be any frame, not just the `parent` or the `child` link.
+We chose the y-axis with respect to the `model` frame so we put `1` in the y element and zeros in the other.
+For the revolute joint we need to define the `<limits>` of our rotation angle in the `<lower>` and `<upper>` tags.
+
+**Note**: The angles are in radians.
+
+#### Right wheel joint
+
+The `right_wheel_joint` is very similar except for the pose of the joint.
+This joint connects the `right_wheel` with the `chassis`.
+
+```xml
+<joint name='right_wheel_joint' type='revolute'>
+    <pose relative_to='right_wheel'/>
+    <parent>chassis</parent>
+    <child>right_wheel</child>
+    <axis>
+        <xyz expressed_in='__model__'>0 1 0</xyz>
+        <limit>
+            <lower>-1.79769e+308</lower>    <!--negative infinity-->
+            <upper>1.79769e+308</upper>     <!--positive infinity-->
+        </limit>
+    </axis>
+</joint>
+```
+
+#### Caster wheel joint
+
+For the caster we need a different type of joint (connection).
+We used `type='ball'` which gives 3 rotational degrees of freedom.
+
+```xml
+<joint name='caster_wheel' type='ball'>
+    <parent>chassis</parent>
+    <child>caster</child>
+</joint>
+```
+
+## Conclusion
+
+Run the world:
+
+`ign gazebo car_demo.sdf`
+
+It should look like this:
+
+![two_wheeled_robot](two_wheeled_robot.png)
+
+Hurray! We build our first robot. You can learn more details about SDFormat tags [here][http://sdformat.org/spec]. In the next [tutorial](../moving_robot_demo.md) we will learn how to move our robot around.
+
+### TODO
+
+* link to moving_robot_demo, world tutorials.
```

Comment:
For now you can remove these lines and any other references to other tutorials; we'll add those all back once everything is on master